rllib/utils/exploration/exploration.py

```diff
@@ -32,6 +32,30 @@ def __init__(self,
         self.worker_index = worker_index
         self.framework = check_framework(framework)
 
+    def before_forward_pass(self,
```

Comment:
Can we call this `before_compute_actions`?

Reply:
Also, could you make the changes to the sampler to call this hook? I think you only need to change sampler.py::_do_policy_eval.

Reply:
I moved it into _build_compute_actions. I think that's better. Then we don't have to change Policy.compute_actions anymore.